src/components/Pricing.jsx

```diff
@@ -1,65 +0,0 @@
-import React from 'react'
```

Comment:
https://github.com/MoveOnOrg/Spoke/blob/master/src/routes.jsx#L23 imports this file. I'd expect that to throw an error if the file is deleted.